From 43c0e0cbb871170ad45b9a0954522e14d9c35823 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Fri, 28 Mar 2014 10:21:39 +0800
Subject: [PATCH 332/602] ENGR00305797 wfd-uibc:change permission for input
 event

Give read and write permission of input event0 and event2 to mediaserver.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 imx6/etc/ueventd.freescale.rc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/imx6/etc/ueventd.freescale.rc b/imx6/etc/ueventd.freescale.rc
index 218e122..19b0b40 100644
--- a/imx6/etc/ueventd.freescale.rc
+++ b/imx6/etc/ueventd.freescale.rc
@@ -22,6 +22,8 @@
 /dev/watchdog             0660   system     root
 /dev/ir                   0660   system     system
 /dev/mma8x5x              0640   compass     system
+/dev/input/event2         0666   root        input
+/dev/input/event0         0666   root        input
 
 # sysfs properties
 /sys/devices/virtual/input/input*   name        0660  root   input
-- 
1.8.0

